app/src/main/res/layout-land/dialog_playback_parameter.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:clickable="false"
+    android:paddingLeft="@dimen/video_item_search_padding"
+    android:paddingTop="@dimen/video_item_search_padding"
+    android:paddingRight="@dimen/video_item_search_padding">
+
+    <RelativeLayout
```

Comment:
While you are at it, please convert everything to a single ConstraintLayout nested inside a ScrollView, it will make everything simpler and with less nesting.